src/helpers/getLivestormConfig.js

```diff
@@ -0,0 +1,80 @@
+const configStore = require('./configStore.js')
+const prompts = require('prompts')
+
+const livestormDomain = require('./livestormDomain')
+
+/**
+ * 
+ * @typedef { import('../../types').Config } LivestormConfig
+ * @typedef { import('../../types').Environment } LivestormEnvironment
+ */
+
+module.exports = async function getLivestormConfig(envName = 'development') {
+  /**
+   * 
+   * @type { (LivestormConfig }
+   */
+  let livestormConfig = null
+
+  try {
+    livestormConfig = require(`${process.cwd()}/livestorm.config.js`)
```

Comment:
I'd rather stick to the current naming `environments.js` and also we need to keep retro-compatibility for a while and support both JSON + JS.
I think we can drop support for havings envs directly inside the package.json though

Reply:
For me it feels like the environments is now a piece of the configuration with the current change. This the conf that includes the environments. Plus it could create a misunderstanding with the `envs` command. 
And finally the `livestorm` prefix shows clearly that this is the config of Livestorm and is easy to recognize and this pattern is used by a lot of CLI libraries. Free to discuss it anytime.